vip-integrations.php

```diff
@@ -23,6 +23,7 @@
 require_once __DIR__ . '/integrations/parsely.php';
 require_once __DIR__ . '/integrations/vip-governance.php';
 require_once __DIR__ . '/integrations/enterprise-search.php';
+require_once __DIR__ . '/integrations/remote-data-blocks.php';
```

Comment:
Needs a temporary file_exists/class_exists check - see jetpack for example.

Reply:
I've added this check in for this line, as well as for loading the integration instance.